Add Copr issue (#178)
packit/status: content/issues/2025-03-05-copr-issue.md

---
title: "Copr SRPM builds failing"
date: 2025-03-05T08:00:00+01:00
affected:
  - Copr
resolved: false
resolvedWhen: 2025-03-05T12:00:00+01:00
section: issue
severity: disrupted
---

> SRPM builds in Copr have been failing, the reason is still unknown, please be patient.

Reviewer: ```suggestion
SRPM builds in Copr are failing, the reason is still unknown, please be patient.
```

Reviewer: Not sure, but it sounds better to me.

> SRPM builds in Copr have been failing, the reason is still unknown, please be patient.

Reviewer: ```suggestion
SRPM builds in Copr are failing, the reason is still unknown, please be patient.
```

Not sure, but it sounds better to me.
